internal/worker/domainservices/manifold.go

```diff
@@ -242,14 +242,14 @@ func (config ManifoldConfig) output(in worker.Worker, out any) error {
 func NewControllerDomainServices(
 	dbGetter changestream.WatchableDBGetter,
 	dbDeleter coredatabase.DBDeleter,
-	objectStoreGetter objectstore.ModelObjectStoreGetter,
+	controllerObjectStoreGetter objectstore.ModelObjectStoreGetter,
```

Comment:
I initially found this a bit confusing, mainly due to the type naming and another name for modelObjectStoreGetter below, but that might just be me.

Suggestion:
Would something like controllerModelObjectStoreGetter make the distinction between controller and other models clearer?

Reply:
Yeah I can appreciate the confusion here. What this is not is the controller models object store but an object store for the entire controller. I think the model assumption comes into play because of the type.

I have fixed this by introducing a new type `object store.NamespacedObjectStoreGetter`. This demonstrates that the object store is fixed onto a given namespace but it is not associated with a model. Hopefully this makes it clearer to the reader what is going on.